import java.lang.Exception;

public class ExceptionTest {
	
	public static void main(String[] args){
		
		int[] arr = new int[]{"one","two","three"};
        for(int i:arr)
        {
                  System.out.println(i);
        }
	}
} 